parallel_integrator_ext.cc fails to build on Windows

```
  Building Custom Rule C:/Users/luc.bourhis/Developer/dials/modules/dials/src/dials/algorithms/integration/CMakeLists.txt
  parallel_integrator_ext.cc
  Please define _WIN32_WINNT or _WIN32_WINDOWS appropriately. For example:
  - add -D_WIN32_WINNT=0x0601 to the compiler command line; or
  - add _WIN32_WINNT=0x0601 to your project's Preprocessor Definitions.
  Assuming _WIN32_WINNT=0x0601 (i.e. Windows 7 target).
C:\Users\luc.bourhis\Developer\dials\modules\dials\src\dials\algorithms\integration\boost_python\parallel_integrator_ext.cc(1,1): error C1128: number of sections exceeded o
bject file format limit: compile with /bigobj [C:\Users\luc.bourhis\Developer\dials\build\dials\src\dials\algorithms\integration\dials_algorithms_integration_parallel_integ
rator_ext.vcxproj]
```

This happened during bootstrapping from scratch.
Microsoft Windows 11 Enterprise 10.0.22631 (22631)
Visual Studio Community Edition 2022, 17.14.3

--- a/installer/bootstrap.py
+++ b/installer/bootstrap.py
@@ -1097,8 +1097,8 @@ conda activate {dist_root}/conda_base
 cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
 project(dials)
 
-if (CMAKE_UNITY_BUILD AND MSVC)
-    # Windows can fail in this scenario because too many objects
+if (MSVC)
+    # Windows can fail because of too many objects
     add_compile_options(/bigobj)
 endif()
 

--- a/newsfragments/3019.bugfix
+++ b/newsfragments/3019.bugfix
@@ -1,0 +1,1 @@
+Fix failed builds on Visual Studio 2022 due to excessively large obj files.

